src/Common/FitOnFhir.Common.Tests/TableRepositoryBaseTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Azure;
+using Azure.Data.Tables;
+using FitOnFhir.Common.Config;
+using FitOnFhir.Common.Models;
+using FitOnFhir.Common.Repositories;
+using FitOnFhir.Common.Resolvers;
+using NSubstitute;
+using NSubstitute.ExceptionExtensions;
+using NSubstitute.Routing.Handlers;
+using Xunit;
+
+namespace FitOnFhir.Common.Tests
+{
+    public abstract class TableRepositoryBaseTests<T>
```

Comment:
We should have a test that ensures non 412 RequestFailedExceptions aren't handled.